### ISSUE TITLE
Option to use other build tools?

### ISSUE THREAD

**Issue by cranesandcaff** [half-pipe/half-pipe#76]
I forked half-pipe with the intention of adding support for GulpJS as well as molding it to fit my own workflow a bit better. Would you be interested in a PR?

**Comment by joefiorini** [comment]
Yes, I'd love to see what you've come up with. I've been wanting to add broccoli support for awhile, but haven't had time. This would be really helpful, thanks!

**Comment by cranesandcaff** [comment]
Right now I basically copied your format for the preprocessor. If you wanted to offer multiple or any preprocessor, maybe a yaml or Json config?

**Comment by cranesandcaff** [comment]
My logic for allowing a Json config is that a lot of developers have their own existing boilerplate for bower/grunt and instead of generating a default automatically offer the choice to import existing package.json and bower and grunt files. Now sure how that would work but I know in the video I believed you mentioned yeoman.

Is something like yeoman an end goal for half pipe?
